### PR TITLE
CURL examples for Neutron

### PR DESCRIPTION
**Pull request by mdarby**

### CODE REVIEW

**Comment by mdarby** [comment]
Any love for this? @smashwilson? :)

File: src/docs/cloud-networks/getting-started/samples/create_port.rst [hunk 1]
```diff
@@ -49,8 +49,8 @@
 
 .. code-block:: sh
 
-  // Not currently supported by this SDK
-
-  $ curl -X GET $ENDPOINT/entities/{entityId}/checks \
-    -H "X-Auth-Token: $TOKEN" \
+  curl -X "POST" "https://dfw.networks.api.rackspacecloud.com/v2.0/ports" \
+    -H "X-Auth-token: $TOKEN" \
```
Comment (smashwilson) [inline src/docs/cloud-networks/getting-started/samples/create_port.rst]: Capitalization: "X-Auth-**T**oken"

I know headers are case-insensitive, but hey.

(You also don't need the quotes around POST.)

File: src/docs/cloud-networks/getting-started/samples/create_security_group.rst [hunk 2]
```diff
@@ -57,8 +57,13 @@
...
-  $ curl -X GET $ENDPOINT/entities/{entityId}/checks \
-    -H "X-Auth-Token: $TOKEN" \
-    -H "Accept: application/json" | python -m json.tool
+  curl -X "POST" "https://dfw.networks.api.rackspacecloud.com/v2.0/security-groups" \
+    -H "X-Auth-token: $TOKEN" \
+    -d $'{
+    "security_group": {
+      "name": "MyNewSecurityGroup",
+      "description": "MyRules",
+      "tenant_id": "$TENANT_ID"
```
Comment (smashwilson) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: I just checked, and I don't believe `$'...'` interpolates in bash.

**Comment by smashwilson** [comment]
@mdarby Comment'd

File: src/docs/cloud-networks/getting-started/samples/create_security_group.rst (hunk 2 above)
Comment (mdarby) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: hrm. The API client I use auto generated these examples. Let me double check :/

Comment (smashwilson) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Huh, weird. This was my test:

```
$ export FOO=bar
$ echo $'something ${FOO} yup'
something ${FOO} yup
```

Comment (mdarby) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Seems that $TOKEN interpolation is all over the docs. Do you think I should update all, or?

Comment (smashwilson) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Er, nuts. I'd say not in this pull request. Maybe fix it here, but put in an issue for the other cases? How does that sounds?

Comment (mdarby) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Sounds like a plan! I've made the other capitalization update.

Comment (mdarby) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Actually, the interpolation works when wrapped in double quotes. :)

Comment (smashwilson) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Ah, you mean like `$" ... "`? Do you have to escape all the double quotes in the JSON too, then?

In general doing JSON in bash is a bit of a pain - I've seen a few approaches used throughout the docs. If we can find one that works cleanly, maybe I'll do a pass and update the rest of them to be consistent.

Comment (mdarby) [inline src/docs/cloud-networks/getting-started/samples/create_security_group.rst]: Copy/pasting a sample of the commands shows that it sends as expected (of course it fails as the dynamic values aren't setup). I'll experiment.

**Comment by mdarby** [comment]
Alrighty; confirmed all cURL commands are GTG.

**Comment by smashwilson** [comment]
:metal: Sweet, thanks! :shipit:

**Pull request merged**